src/forms/subscriptions.ts

```diff
@@ -405,4 +453,17 @@ class FormFields extends BaseFormFields {
             return option.value === this.getCurrentChannelID();
         };
     }
+
+    getMaxTitleNameLength(): number {
```

Comment:
Should be cleaner if we move the const out and remove the function.

Reply:
Sure. Moved to the constants file.  done.